.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
```

Comment:
Hm, I was thinking about using `.git/hooks/pre-commit` right away rather then using a dependency for now.